components/landing_page.module.css

```diff
@@ -0,0 +1,135 @@
+/* Wrapper for landing page content */
+.body {
+	margin-right: auto;
+	margin-left: auto;
+	max-width: 1440px;
+	padding-right: 100px;
+	padding-left: 100px;	
+}
+
+/* General Column */
+.column {
+	display: flex;
+	flex-direction: column;
+	flex-basis: 100%;
+	flex: 1;
+	justify-content: center;
+	align-items: center;
+  padding-right: 30px;
+  padding-left: 30px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+
+/* General Row */
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 100%;
+  justify-content: center;
+} 
+
+/* Main Text */
+.TCFG {
+  width: 715px;
+  height: 184px;
+  font-family: GujaratiSangamMN;
+  font-size: 72px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: center;
+  color: #000000;
+  padding-bottom: 30px;
+}
+
+/* Description text */
+.text {
+  font-size: 20px;
+  min-width: 300px;
+  padding-bottom: 40px;
+  font-family: HiraginoSans-W1;
+  text-align: justify;
+  text-justify: inter-word;
+}
+
+/* 'Learn More' button */
+.Rectangle {
+  width: 298px;
+  height: 84px;
+  border-radius: 30px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
+  background-color: #d8d8d8;
+}
+
+
+/* Text within button */
+.LearnMore {
```

Comment:
This might be better as an `id` instead of a class: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

This could also apply to`TCFG`